version.gradle

```diff
@@ -1,3 +1,3 @@
 ext {
-    sdk_version = '0.0.0'
+    sdk_version = '0.0.6'
```

Comment:
Why's this here?

Reply:
To release a new version manually, cause we have to fix the script for get the new tag